### PR TITLE
Dresean branch

### PR DESCRIPTION
**Pull request by dresean**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
through postman w/ minor validation from server and front end.
- [x] Test B
through the server via api calls from the front end on the production build
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by EricGomez101** [comment]
Tested; found no errors, syntax is looking good. Ship it!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (20djm) [inline .vscode/settings.json]: Is it possible to uncommit this?

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "postinstall": "cd react_django/react-src && npm run build",
-    "postbuild": "mv build ../django-src/"
+    "postbuild": "rm -rf ../django-src/build && mv build ../django-src/"
   },
```
Comment (20djm) [inline package.json]: Is it possible to uncommit this?

File: react_django/django-src/Pipfile.lock [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (20djm) [inline react_django/django-src/Pipfile.lock]: Is it possible to uncommit this?

**Pull request closed without merging**